app/controllers/admin/companies_controller.rb

```diff
@@ -23,7 +23,7 @@ def edit; end
 
   def update
     if @company.update(company_params)
-      redirect_to admin_companies_url, notice: '企業を更新しました。'
+      redirect_to company_url, notice: '企業を更新しました。'
```

Comment:
[imo] 以下の形はいかがでしょうか。
```suggestion
      redirect_to @company, notice: '企業を更新しました。'
```

Reply:
@reckyy 
ありがとうございます。`@company`に修正しました。
[リダイレクトするパスをシンプルなものに変更](https://github.com/fjordllc/bootcamp/pull/7621/commits/0f5d77244ad60b15de7695cba9b57477a975c4bc)

Reply:
ありがとうございます！
ちなみに、何故 `company_url`と記述されたのでしょうか。（ `company_url(@company)`ではなく）（全く責めているわけではなく、自身の知らない記法があるかもとお伺いしたいだけです。）
https://api.rubyonrails.org/v7.1.2/classes/ActionController/Redirecting.html#method-i-redirect_to

Reply:
@reckyy 
こちらはルーティングを確認して引数を指定し忘れました。
```
company GET    /companies/:id(.:format)    companies#show
```